pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -210,6 +214,7 @@ func (f *framework) RunPreFilterPlugins(
 			klog.Error(msg)
 			return NewStatus(Error, msg)
 		}
+		klog.V(10).Infof("running %q prefilter plugin for pod %q successfully", pl.Name(), pod.Name)
```

Comment:
it's easier to just log the status in my opinion. This makes sure the status is always logged and you don't need to worry about whether the status is success or not. Makes the control flow easier.

Reply:
Added status message.